akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphFlexiMergeSpec.scala

```diff
@@ -272,11 +270,11 @@ class GraphFlexiMergeSpec extends AkkaSpec {
       }
 
       val p = g.run()
-      val s = SubscriberProbe[String]
+      val s = TestSubscriber.probe[String]
       p.subscribe(s)
       val sub = s.expectSubscription()
       sub.request(100)
-      (s.probe.receiveN(2).map { case OnNext(elem) ⇒ elem }).toSet should be(Set("((+,b),f)", "((+,a),e)"))
+      (for (_ ← 1 to 2) yield s.expectNext()).toSet should be(Set("((+,b),f)", "((+,a),e)"))
```

Comment:
There is already a `def expectNext(e1: I, e2: I, es: I*)` which expects multiple elements in order.